.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# We write CODEOWNERS files in terms of Github teams.
+# When adding (or removing) code owners to the project, remember to update the `developers` within the pom.xml.
+
+/.github/ @kroxylicious/admins
+* @kroxylicious/developers
```

Comment:
does it matter that the team is `Developers` with capital D?

Reply:
I see lowercase is working fine in the kroxylicious repo.

Reply:
yeah, I noticed if when you tag in a comment, Github generates the lower case form.

<img width="699" alt="Screenshot 2025-03-07 at 09 18 12" src="https://github.com/user-attachments/assets/5a6614d2-fbde-4ac1-893e-9b2dc14ac225" />